hackathon_site/hardware/api_filters.py

```diff
@@ -24,6 +25,13 @@ class IncidentFilter(filters.FilterSet):
     hardware_id = filters.NumberFilter(field_name="order_item__hardware__id")
     team_id = filters.NumberFilter(field_name="order_item__order__team__id")
 
+class OrderItemFilter(filters.FilterSet):
+    queryset = OrderItem
+    serializer_class = OrderItemListSerializer
+
+    hardware_id = filters.NumberFilter(field_name="order__id")
+    team_id = filters.NumberFilter(field_name="order__team_code")
```

Comment:
call this team_code instead of team_id and team_code is a char field so use CharFilter instead of NumberFilter. Also, team is nested in order, not team_code so change the field name to "order__team__team_code"